cpp/src/link_prediction/jaccard_impl.cuh

```diff
@@ -27,19 +27,13 @@ struct jaccard_functor_t {
   template <typename weight_t>
   weight_t __device__ compute_score(weight_t cardinality_a,
                                     weight_t cardinality_b,
-                                    weight_t cardinality_a_intersect_b) const
+                                    weight_t cardinality_a_intersect_b,
+                                    weight_t cardinality_a_union_b) const
   {
-    return cardinality_a_intersect_b / (cardinality_a + cardinality_b - cardinality_a_intersect_b);
-  }
-};
-
-struct weighted_jaccard_functor_t {
-  template <typename weight_t>
-  weight_t __device__ compute_score(weight_t weight_a,
-                                    weight_t weight_b,
-                                    weight_t min_weight_a_intersect_b) const
-  {
-    return min_weight_a_intersect_b / (weight_a + weight_b - min_weight_a_intersect_b);
+    return (fabs(static_cast<double>(cardinality_a_union_b) - double{0}) <
+            double{2} / double{1 << 30})
```

Comment:
What is `double{2} / double{1 << 30}`? Better have some comments describing what this is.

Reply:
And why are you subtracting `double{0}`? Isn't `cardinality_a_union_b - 0` == `cardinality_a_union_b`? What am I missing here?

Reply:
Fixed.

Reply:
Fixed.